tool/ood-gen/lib/job.ml

```diff
@@ -5,6 +5,29 @@ let all () =
   Utils.yaml_sequence_file of_yaml "jobs.yml"
   |> List.sort (fun j1 j2 -> compare (job_date j2) (job_date j1))
 
+module JobFeed = struct
+  let create_entry (job : t) =
+    let content = 
+      Syndic.Atom.Text (Printf.sprintf "Company: %s\nLocations: %s\n" job.company (String.concat ", " job.locations)) 
+    in
+    let id = Uri.of_string (job.link) in
+    let authors = (Syndic.Atom.author "Ocaml.org", []) in
+    let updated = match job.publication_date with
+      | Some date_str -> Syndic.Date.of_rfc3339 (date_str ^ "T00:00:00-00:00")
+      | None -> Syndic.Date.of_rfc3339 "1970-01-01T00:00:00Z" in
```

Comment:
I think the jobs are being filtered out because the default date is more than 365 days in the past. Since we don't have so many jobs that we'd need to limit, probably best to not have a cutoff, and also assume that any job that doesn't have a date set is very recent

Reply:
Nice, wasn't sure what that was doing. Thank you ❤️